src/test/java/uk/nhs/tis/trainee/notifications/service/EmailServiceIntegrationTest.java

```diff
@@ -116,11 +139,12 @@ void shouldIncludeContentInTemplates(String template) throws Exception {
 
   @ParameterizedTest
   @MethodSource("getEmailTemplates")
-  void shouldGreetExistingUsersConsistentlyWhenNameNotAvailable(String template) throws Exception {
+  void shouldGreetExistingUsersConsistentlyWhenNameNotAvailable(NotificationType notificationType,
```

Comment:
I guess these tests would break if a subsequent version of a template used another fragments.greeting version which included a different salutation. But that is a problem for then, I imagine!